diff -urN srt-repo.orig/CMakeLists.txt srt-repo/CMakeLists.txt
--- srt-repo.orig/CMakeLists.txt	2025-12-31 17:52:15
+++ srt-repo/CMakeLists.txt	2025-12-31 17:52:43
@@ -7,7 +7,7 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 #
 
-cmake_minimum_required (VERSION 2.8.12 FATAL_ERROR)
+cmake_minimum_required (VERSION 3.5.0 FATAL_ERROR)
 set (SRT_VERSION 1.5.4)
 
 set (CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/scripts")
